ui/k-toolbar/main.kytos

```diff
@@ -58,6 +58,11 @@
                     tooltip="Enter the southbound priority"
                     placeholder="southbound priority" icon="arrow-right"></k-input>
 
+            <k-input id="max_paths" v-model:value="max_paths"
+                    title="Paths to consider"
```

Comment:
The title wasn't immediately clear to me that this is related to max paths:

![20250410_104443](https://github.com/user-attachments/assets/05a2eacb-0c7c-4d63-bbb3-be9410e29aac)

How can we improve this? Maybe `"Max paths to compute|consider"?` or something similar?

Reply:
Change it to `Max paths to compute|consider`. Difficult to fit more info.